oggm/sandbox/distribute_2d.py

```diff
@@ -206,14 +207,18 @@ def assign_points_to_band(gdir, topo_variable='glacier_topo_smoothed',
 
 
 @entity_task(log, writes=['gridded_data'])
```

Comment:
I think this should be `'gridded_simulation'` instead of `'gridded_data'` now

Reply:
Good catch! Fixed now. Also added more variables to the file so that it can be used by the 3dviz tool (to no success so far)